pkg/values/big_int.go

```diff
@@ -31,9 +30,10 @@ func (b *BigInt) Unwrap() (any, error) {
 
 func (b *BigInt) UnwrapTo(to any) error {
 	if b == nil || b.Underlying == nil {
-		return errors.New("could not unwrap nil values.BigInt")
+		return fmt.Errorf("could not unwrap nil")
```

Comment:
The linter should be telling you to use `errors.New` actually.
```suggestion
		return errors.New("could not unwrap nil")
```